packages/terra-date-picker/src/react-datepicker/date_utils.js

```diff
@@ -319,30 +300,6 @@ export function isDayDisabled (day, { minDate, maxDate, excludeDates, includeDat
     false
```

Comment:
[addMinutes() ](https://github.com/cerner/terra-framework/blob/6623edd64551db6f735b3854956149e90fc082f0/packages/terra-date-picker/src/react-datepicker/date_utils.js#L170) function can also be removed. since it was referred only in Time.jsx

Reply:
Done - https://github.com/cerner/terra-framework/pull/875/commits/dbf858ea4fd07d28f4d79988fa50e22a0f744054.